### ISSUE TITLE
Add option to disable right-click target with MQMap

### ISSUE THREAD

**Issue by HK47196** [macroquest/macroquest#784]
**Is your feature request related to a problem? Please describe.**
MQMap has no way to disable right-click label targeting. May be used unknowingly exposing risk and/or used accidentally. 

**Describe the solution you'd like**
Add option to opt-out of right-click label targeting on map.

**Additional context**

See: #298 

If /eqtarget will be the default, perhaps right-click label targeting should be an opt-in rather than opting out of it.

**Comment by dannuic** [comment]
This is something we've talked about before, and I totally agree -- but there was a good deal of pushback making this behavior the default because it changes long-standing functionality. Hopefully more people can weigh in here and come up with suggestions. I personally think having options for either mqtarget and eqtarget is a good solution.

**Comment by projecteon** [comment]
This request seems to ask for it as an opt-out though making the default functionality stay as is.
Simplest might be an ini toggle for which target method to use with default mqtarget?

**Comment by brainiac** [comment]
I feel like the obvious fix here is to just make it use the same click configuration as everything else.

**Comment by dannuic** [comment]
Yeah, definitely, but I think the question is what default do we ship with (also I think eqtarget will still generate packets when out of range and simply fail, we'd have to make sure there is a check in the command)

**Comment by brainiac** [comment]
it would have to be mqtarget, since map targets by id

**Comment by brainiac** [comment]
https://github.com/macroquest/macroquest/blob/master/src/plugins/map/MQ2Map.cpp#L50-L88

https://docs.macroquest.org/plugins/core-plugins/mq2map/mapclick/
